api/src/types.rs

```diff
@@ -78,15 +78,22 @@ pub struct Status {
 	pub connections: u32,
 	// The state of the current fork Tip
 	pub tip: Tip,
+	// The current sync status
+	pub sync_status: String,
```

Comment:
If this is `sync` vs `synced` then maybe this is just `status` and not `sync_status`?
I'm not sure if we are exposing the actual "sync status" here or exposing a "status" that we are using our internal sync_status to derive from.

Reply:
The reason I'm being pedantic here is from the original issue raised about this.

"Expose the `Basic Status > Current Status: Running` field in the Node API"

I'd argue nobody actually cares about the sync status itself - they just care about when the node is finally "running".

Reply:
Changed to `status` with two possible status: syncing or running.